volumes/miovision/sql/create-table-volumes.sql

```diff
@@ -1,129 +1,60 @@
 ﻿-- Table: miovision_api.volumes
-
 -- DROP TABLE miovision_api.volumes;
-
 CREATE TABLE miovision_api.volumes
 (
-  volume_uid bigserial NOT NULL,
-  intersection_uid integer,
-  datetime_bin timestamp without time zone,
-  classification_uid integer,
-  leg text,
-  movement_uid integer,
-  volume integer,
-  volume_15min_mvt_uid integer,
-  CONSTRAINT volumes_volumes_15min_mvt_uid_fkey FOREIGN KEY (volume_15min_mvt_uid)
-      REFERENCES miovision_api.volumes_15min_mvt (volume_15min_mvt_uid) MATCH SIMPLE
-      ON UPDATE NO ACTION ON DELETE SET NULL
+    volume_uid bigserial NOT NULL,
+    intersection_uid integer,
+    datetime_bin timestamp without time zone,
+    classification_uid integer,
+    leg text,
+    movement_uid integer,
+    volume integer,
+    volume_15min_mvt_uid integer,
+    CONSTRAINT volumes_intersection_uid_datetime_bin_classification_pkey 
+    PRIMARY KEY (intersection_uid, datetime_bin, classification_uid, leg, movement_uid),
+    CONSTRAINT volumes_volume_15min_mvt_uid_fkey FOREIGN KEY (volume_15min_mvt_uid)
```

Comment:
we will have to get of this foreign key then :cry_cat:

Reply:
done!